docs/application/native/guides/internationalization/i18n.md

```diff
@@ -286,7 +286,7 @@ The enumeration object enables navigation through the enumeration values, with t
 You can get the number of values stored in the enumeration object with the `i18n_uenumeration_count()` function.
 
 <a name="ulocale"></a>
-## Locale-sensitive Operations with Ulocale
+## Locale-Sensitive Operations with Ulocale
 
 The Ulocale API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__ULOCALE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__ULOCALE__MODULE.html) applications) represents a specific geographical, political, or cultural region. Locale-sensitive operations [use the Ulocale functions to tailor information](#locales) for the user. For example, displaying a number is a locale-sensitive operation. The number must be formatted according to the customs and conventions of the user's native country, region, or culture.
```

Comment:
Please can you change the following:
Line 121 :
**Insufficient information**The calendar uses the default information to specify the missing fields. This can vary by calendar: for the Gregorian calendar, the default for a field is the same as that of the start of the epoch, such as `I18N_UCALENDAR_YEAR = 1970`, `I18N_UCALENDAR_MONTH = JANUARY`, `I18N_UCALENDAR_DATE = 1`.



Please can you change to:
**Insufficient information**: The calendar uses the default information to specify the missing fields. This can vary by calendar. For example, the Gregorian calendar, the default for a field is the same as that of the start of the epoch, such as `I18N_UCALENDAR_YEAR = 1970`, `I18N_UCALENDAR_MONTH = JANUARY`, `I18N_UCALENDAR_DATE = 1`.

Line 122:
- **Inconsistent information**If the fields conflict, the calendar prefers the most recently set fields. For example, when determining the day, the calendar looks for one of the following field combinations listed in the following table. The most recent combination, as determined by the most recently set single field, is used.

Please change as:

- **Inconsistent information**: If there is field’s conflict, the calendar prefers the most recently set fields. For example, when determining the day, the calendar looks for one of the following field combinations listed in the following table. The most recent combination, as determined by the most recently set single field, is used.

Line 295 to 297:
Please can you add colon after “`newLanguage`”, “`newLanguage + newCountry`”, “`newLanguage + newCountry + newVariant`” words.

Line 297:
there are 2 variants
Please change “2” to “two”

Line 323:
The set supports 2 functions:
Please change “2” to “two”

Line 350:
Ranges are indicated by placing a '-' between 2 characters
Please change “2” to “two”

Line 372: 
The following table provide examples of formal syntax patterns.
Please change . with : at the end of the sentence.

Line 421:
between the other 2 scripts.

Please change “2” with “two”

Line 435:
The field position within formatted output is tracked with 2 indices: the indices of the first and the last characters of the field.
Please change to:
The field position within formatted output is tracked with two indices, which includes the indices of the first and the last characters of the field.

Line 459: 
The following table lists the supported measures and units.

Please add colon at the end of the sentence.

Line 1460 :
To check whether the Ucalendar date
Please change “check” to “verify”

Line 1909, line 1915, line 1921:

Please change “check” to “verify”

Line 1960:

Please change “check” to “verify”